api/src/test/java/org/openmrs/OrderEntryIntegrationTest.java

```diff
@@ -140,6 +140,8 @@ public void shouldPlaceATestOrder() {
 		order.setFrequency(orderService.getOrderFrequency(3000));
 		order.setSpecimenSource(conceptService.getConcept(22));
 		order.setNumberOfRepeats(3);
+		order.setFulfillerStatus(Order.FulfillerStatus.RECEIVED);
+		order.setFulfillerComment("A comment from the filler");
```

Comment:
@wluyima I added the test case to the _shouldPlaceATestOrder_ test  test method. Is this sufficient? 

I guess DrugOrder is a bit more specific than TestOrder. So I choose this method instead

Reply:
This is fine too